smartapps/smartthings/virtual-device-manager.src/virtual-device-manager.groovy

```diff
@@ -15,7 +15,7 @@
  */
 
 definition(
-    name: "Virtual Device Manager",
+    name: "Virtual Device Creator",
```

Comment:
@workingmonk Is it possible to change the filename so that it matches the new name?